### PR TITLE
90%: Bump blueprint_rb version

### PR DESCRIPTION
**Pull request by rsinger**
Add support for multiple node query and assetNode filters to Talis::Hierarchy

### CODE REVIEW

**Comment by rsinger** [comment]
I guess the question here is whether or not we feel this needs tests - I think our decision was that we would use talis_rb as the test suite for blueprint_rb and metatron_rb, which, in turn, makes me feel like this needs tests for `filter[asset.node]`.

The bit I'm not sure about it is that Blueprint version we're updating to also supports searching on Assets, which Talis::Hierarchy::Asset currently doesn't handle.

@pads what's your take on this?

**Comment by pads** [comment]
I think, at the very least for documentations sake, there should be a test for the filter. Ideally in future these might be more domain-driven than API-driven.

**Comment by rsinger** [comment]
Ok.

File: spec/integration/fixtures/node_asset_relns_assets.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
+{
+  "zzz": {
+    "type": "coretext_preferences_awaiting",
```
Comment (pads) [inline spec/integration/fixtures/node_asset_relns_assets.json]: Not sure this domain-specific stuff should be in a public library.

File: spec/integration/hierarchy/node_spec.rb [hunk 2]
```diff
@@ -750,6 +752,86 @@
     end
   end
 
+  context 'searching asset.node relationships' do
```
Comment (pads) [inline spec/integration/hierarchy/node_spec.rb]: 'searching for nodes related by assets'

**Pull request merged**